tests/runtests.pl

```diff
@@ -2877,11 +2948,12 @@ sub checksystem {
     my $hosttype=join(' ', runclientoutput("uname -a"));
 
     logmsg ("********* System characteristics ******** \n",
-    "* $curl\n",
-    "* $libcurl\n",
-    "* Features: $feat\n",
-    "* Host: $hostname",
-    "* System: $hosttype");
+            "* $curl\n",
+            "* $libcurl\n",
+            "* Features: $feat\n",
+            "* Disabled: $dis\n",
```

Comment:
This can be used uninitialized according to the buildfarm: https://curl.haxx.se/dev/log.cgi?id=20190527165744-10382#prob1

Reply:
@dago  how come these builds run on PR branch like this? The autobuilds are *supposed* to run on master... In this case of course it turned out useful since the solaris builds in the CI didn't spot this problem but the autobuilds one did!

Reply:
@bagder The builds are actually triggered by the GitHub hook and that one seems to be configured to pick up pull requests too. I vaguely remember I changed this from separate triggers (one poll, one trigger) to make the code easier and this looks like an artifact from that. Is the current configuration acceptable? It should be possible to set up a branch filter for autobuilds if needed.